sqlglot/dialects/bigquery.py

```diff
@@ -215,6 +215,12 @@ def _unix_to_time_sql(self: BigQuery.Generator, expression: exp.UnixToTime) -> s
     return ""
 
 
+def _parse_time(args: t.List) -> exp.Func:
+    if len(args) == 1:
+        return exp.TsOrDsToTime(this=args[0])
+    return exp.Anonymous(this="TIME", expressions=args)
```

Comment:
Decided to use this as a fallback so that the scope of the PR is smaller and the patch is more conservative. It's not straightforward to transpile all variants so I thought we can start with the single argument one and iterate from there if ever needed..